backend/python/app/rest/user_routes.py

```diff
@@ -113,7 +113,7 @@ def create_user():
 
 
 @blueprint.route("/<int:user_id>", methods=["PUT"], strict_slashes=False)
```

Comment:
can we restrict `create_user()` and `delete_user()` endpoints to `Admin`? This may or may not break some tests, not sure.

Reply:
what about the get_users() and update_user() endpoints? I think the update_user() should be admin only, since its updating the users role but not sure about get_users()

Reply:
ur right but i think that's out of scope for this pr